specification/containerservice/Fleet.Management/tspconfig.yaml

```diff
@@ -2,7 +2,9 @@ emit:
   - '@azure-tools/typespec-autorest'
 options:
   '@azure-tools/typespec-autorest':
-    azure-resource-provider-folder: '{cwd}/../resource-manager'
+    arm-types-dir: "{project-root}/../../common-types/resource-management"
```

Comment:
@timotheeguerin, @markcowl: The change from
```
arm-types-dir: ../../../../../../common-types/resource-management
```
to:
```
arm-types-dir: "{project-root}/../../common-types/resource-management"
```

Is causing a diff in the generated swagger (extra `..`):

```
-            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+            "$ref": "../../../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
```

https://dev.azure.com/azure-sdk/public/_build/results?buildId=2788701&view=logs&j=9cd39ef8-47c1-5b58-08db-8c3708712f1b&t=462145f3-d740-50de-51e9-126e7c0a29e2&l=43

Is the new path correct or incorrect?